@here/harp-mapview/lib/MapView.ts

```diff
@@ -1471,11 +1476,29 @@ export class MapView extends THREE.EventDispatcher {
 
     /**
      * Get distance from camera to the point of focus in world units.
+     *
+     * @note If camera does not point to any ground anymore the last focus point distance is
+     * then returned.
+     *
+     * @returns Last known focus point distance.
      */
     get lookAtDistance(): number {
```

Comment:
Why do we even expose this? I think we should only expose zoom

Reply:
This value is there for a long time, it allows for distance scaling computations, used in TextElementsRenderer, allows to mimic perspective for 2d (screen space) rendered objects.

Reply:
Okay, but then we can pass it into the TextElementsRenderer. No need to have a public API for it.
I know it's not your code but we should remove it before people start using it and we never get rid of it.

Reply:
That may be quite problematic, because we need to pass it every frame, do you want me to prepare special setter for it (in TextElementsRenderer), it takes all up-to-date MapView params via MapViewState, and this state is not frame persistent, but rather takes everything on demand from MapView, so it is kind of breaking entire interface/data-flow for single param. Please check it out, if it is worth doing. Maybe we could just @hide it in MapView interface?